code/_onclick/item_attack.dm

```diff
@@ -25,7 +25,13 @@
 	I.attack(src, user)
 
 /mob/living/proc/attacked_by(obj/item/I, mob/living/user, def_zone)
-	apply_damage(I.force, I.damtype, def_zone)
+	var/damage = I.force
+	if(mind && mind.demoninfo && mind.demoninfo.banetype == BANEWHITECLOTHES) // Sorry for the snowflake check.
+		if(istype(user, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = user
+			if(istype(H.w_uniform, /obj/item/clothing/under/color/white))
+				damage *= 3
+	apply_damage(damage, I.damtype, def_zone)

```

Comment:
if i'm reading this right, what you could do is make white clothing have a /hit_reaction() like the armor in #13522
unless what you're trying to do is make it so people with white clothes do more damage to demons. then i have no idea